src/components/profile/components/userRecordTables/UserAnnotationRecordsTable.tsx

```diff
@@ -0,0 +1,79 @@
+/* Import Dependencies */
+import { Row, Col } from "react-bootstrap";
+
+/* Import Config */
+import UserAnnotationRecordsTableConfig from "app/config/table/UserAnnotationRecordsTableConfig";
+
+/* Import Hooks */
+import { usePagination } from "app/Hooks";
+
+/* Import API */
+import GetCreatorAnnotations from "api/annotation/GetCreatorAnnotations";
+
+/* Import Components */
+import { Paginator } from "components/elements/Elements";
+import { DataTable } from "components/elements/customUI/CustomUI";
+
+
+/* User annotation record type */
+type DataRow = {
+    jsonPath: string,
+    motivation: string,
+    value: string | number | boolean
+};
+
+
+/**
+ * Component that renders the user annotation records table on the profile page
+ * @returns JSX Component
+ */
+const UserAnnotationRecordsTable = () => {
+    /* Base variables */
+    const { columns } = UserAnnotationRecordsTableConfig();
+    const tableData: DataRow[] = [];
+
+    const pagination = usePagination({
+        pageSize: 25,
+        resultKey: 'userAnnotations',
+        Method: GetCreatorAnnotations
+    });
+
+    pagination.records.forEach(userAnnotation => {
+        let jsonPath: string;
+
+        if (userAnnotation["oa:hasTarget"]["oa:hasSelector"]?.["@type"] === 'ods:ClassSelector') {
+            jsonPath = userAnnotation["oa:hasTarget"]["oa:hasSelector"]["ods:class"];
+        } else if (userAnnotation["oa:hasTarget"]["oa:hasSelector"]?.["@type"] === 'ods:FieldSelector') {
```

Comment:
this has changed to `TermSelector` fyi

Reply:
In the 0.4.0 yes, but this is still in 0.3.0, but thanks for the heads up